Documentation/network/servicemesh/ingress.rst

```diff
@@ -58,39 +58,50 @@ Supported Ingress Annotations
 #############################
 
 .. list-table::
-   :widths: 25 25 50
    :header-rows: 1
 
    * - Name
      - Description
      - Default Value
    * - ``ingress.cilium.io/loadbalancer-mode``
-     - The loadbalancer mode for the ingress. Applicable values are ``dedicated`` and ``shared``.
-     - Defaults to Helm option ``ingressController.loadbalancerMode`` value.
+     - | The loadbalancer mode for the ingress.
+       | Allows a per ingress override
+       | of the default set in the Helm value
+       | ``ingressController.loadbalancerMode``.
+       | Applicable values are ``dedicated`` and
+       | ``shared``.
+     - | ``dedicated``
+       | (from Helm chart)
    * - ``ingress.cilium.io/service-type``
-     - The Service type for dedicated Ingress. Applicable values are ``LoadBalancer`` and ``NodePort``.
-     - Defaults to ``LoadBalancer`` if unspecified.
+     - | The Service type for dedicated Ingress.
+       | Applicable values are ``LoadBalancer``
+       | and ``NodePort``.
+     - ``LoadBalancer``
    * - ``ingress.cilium.io/insecure-node-port``
-     - The NodePort to use for the HTTP Ingress. Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``.
-     - If unspecified, a random NodePort will be allocated by kubernetes.
+     - | The NodePort to use for the HTTP Ingress.
+       | Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``. If unspecified, a random
+       | NodePort will be allocated by kubernetes.
+     - unspecified
    * - ``ingress.cilium.io/secure-node-port``
-     - The NodePort to use for the HTTPS Ingress. Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``.
-     - If unspecified, a random NodePort will be allocated by kubernetes.
+     - | The NodePort to use for the HTTPS Ingress.
+       | Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``. If unspecified, a random
+       | NodePort will be allocated by kubernetes.
+     - unspecified
    * - ``ingress.cilium.io/tcp-keep-alive``
      - Enable TCP keep-alive
-     - 1 (enabled)
+     - ``enabled``
```

Comment:
would it make sense to explicitly call out available values (enabled/disabled), like you do above?

Reply:
Valid point! Added and updated. Thanks for the input.

![image](https://github.com/cilium/cilium/assets/5549063/1355baf0-a6a1-405d-bf4e-49fbaa12b776)